src/message/fetchActions.js

```diff
@@ -448,13 +448,13 @@ export async function tryFetch<T>(
  * (`SearchMessagesScreen`).
  */
 export const doInitialFetch = (): ThunkAction<Promise<void>> => async (dispatch, getState) => {
-  dispatch(registerStart());
   const auth = getAuth(getState());
 
   let initData: InitialData;
```

Comment:
This declaration should go immediately before the `try` where we initialize the variable. (Logically I'd like to think of it as a `const`, where the declaration and initializer are inseparable.) The `registerStart` can be the line before that.